.env.example

```
@@ -1,2 +1,3 @@
 SERVER_ADDR=dns:localhost:5230
-BOT_TOKEN=your_bot_token
\ No newline at end of file
+BOT_TOKEN=your_bot_token
+MEMOS_ADDR=https://memos.example.com
```

How about instance url?

```suggestion
INSTANCE_URL=https://memos.example.com
```